Documentation/network/kubernetes/ipam.rst

```diff
@@ -10,10 +10,19 @@
 Configuring IPAM Modes
 **********************
 
+Cilium supports multiple IP Address Management (IPAM) modes to meet the needs of different environments and cloud providers. 
+
+The following sections provide documentation for each supported IPAM mode:
+
 .. toctree::
    :maxdepth: 1
-   :glob:
 
-   ipam-crd
-   ipam-cluster-pool
-   ipam-multi-pool
```

Comment:
I think these lines can all stay? It seems like they shouldn't be removed.

If there are duplicates vs. the corresponding concepts page, you can keep the network/kubernetes reference and remove the concepts/ipam copy. For instance for multi-pool we don't need to link both pages.